frameworks/Go/goji/src/goji/server.go

```diff
@@ -0,0 +1,220 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"encoding/json"
+	"flag"
+	"runtime"
+	"database/sql"
+	"log"
+	"math/rand"
+	"strconv"
+	"html/template"
+	"sort"
+
+	_ "github.com/go-sql-driver/mysql"
+
+	"github.com/zenazn/goji"
+	"github.com/zenazn/goji/web"
+)
+
+const (
+	// Database
+	connectionString   = "benchmarkdbuser:benchmarkdbpass@tcp(localhost:3306)/hello_world"
+	worldSelect        = "SELECT id, randomNumber FROM World WHERE id = ?"
+	worldUpdate        = "UPDATE World SET randomNumber = ? WHERE id = ?"
+	fortuneSelect      = "SELECT id, message FROM Fortune;"
+	worldRowCount      = 10000
+	maxConnectionCount = 256
+
+	helloWorldString = "Hello, World!"
+	extraFortuneMessage = "Additional fortune added at request time."
+)
+
+var (
+	// Templates
+	tmpl = template.Must(template.
+		ParseFiles("templates/layout.html",
+			       "templates/fortune.html"))
+
+	// Database
+	worldStatement   *sql.Stmt
+	fortuneStatement *sql.Stmt
+	updateStatement  *sql.Stmt
+)
+
+type Message struct {
+	Message string `json:"message"`
+}
+
+type World struct {
+	Id           uint16 `json:"id"`
+	RandomNumber uint16 `json:"randomNumber"`
+}
+
+func randomRow() *sql.Row {
+	return worldStatement.QueryRow(rand.Intn(worldRowCount) + 1)
+}
+
+type Fortune struct {
+	Id      uint16 `json:"id"`
+	Message string `json:"message"`
+}
+
+type Fortunes []*Fortune
+
+func (s Fortunes) Len() int {
+	return len(s)
+}
+
+func (s Fortunes) Swap(i, j int) {
+	s[i], s[j] = s[j], s[i]
+}
+
+type ByMessage struct {
+	Fortunes
+}
+
+func (s ByMessage) Less(i, j int) bool {
+	return s.Fortunes[i].Message < s.Fortunes[j].Message
+}
+
+// Sets the content type of response. Also adds the Server header.
+func setContentType(w http.ResponseWriter, contentType string) {
+	w.Header().Set("Server", "Goji")
+	w.Header().Set("Content-Type", contentType)
+}
+
+// Test 1: Json Serialization
+func serializeJson(c web.C, w http.ResponseWriter, r *http.Request) {
+	setContentType(w, "application/json")
+    json.NewEncoder(w).Encode(&Message{helloWorldString})
```

Comment:
I did not. My gosublime was giving me trouble. I installed gofmt from the command line and applied it. Thanks for the catch.